tox.ini

```diff
@@ -19,10 +19,16 @@ setenv =
     AWS_ACCESS_KEY_ID=foo
     AWS_SECRET_ACCESS_KEY=bar
     C7N_VALIDATE=true
+whitelist_externals=rm
+
+[cleanup]
+commands =
+    rm {homedir}/.cache/cloud-custodian.cache
 
 [testenv:py27]
 commands =
     py.test --tb=native -n auto tests tools {posargs}
+    {[cleanup]commands}
```

Comment:
hmmm... this one seems strange. are tests leaking into the cache, they should be explicitly using a temp file cache per the common test infrastructure. the ci environment runs clean per build as docker containers (travis and drone). where you seeing something cached results here?